<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentScore</class>
 <widget class="QMainWindow" name="StudentScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentScore</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSum">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutR">
        <item>
         <widget class="QPushButton" name="InsertButton">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutL">
        <item>
         <widget class="QLineEdit" name="insertlineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="deletelineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="findlineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
